games-arcade/ccatch/ccatch-5.0.0.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="Help a cat catch birds"
+DESCRIPTION="cCATch is a portation of the game CATch, written in c using SDL"
+HOMEPAGE="https://www.github.com/TheZeldakatze/c-CATch"
+COPYRIGHT="2020 Victor Gamper"
+LICENSE="GNU GPL v3"
+REVISION="1"
+srcGitRev="018c0fd7184e02992488a95ce514c3f1d77d7604" 
+# 6 Dec 2021 commit.
+SOURCE_URI="https://github.com/TheZeldakatze/c-CATch/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="c5f0d7b696d641b6168b92b05e822f76c92de31c784739d94f901b5596e593c5"
+SOURCE_FILENAME="c-CATch-$srcGitRev.tar.gz"
+SOURCE_DIR="c-CATch-$srcGitRev"
+ADDITIONAL_FILES="ccatch.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	cCATch = $portVersion
```

Comment:
PROVIDES should match the recipe name (you can check this if you are running `haikuporter -S`), also missing $secondaryArchSuffix
```diff
- 	cCATch = $portVersion
+ 	ccatch$secondaryArchSuffix = $portVersion
```